core/dbt/adapters/base/relation.py

```diff
@@ -1,13 +1,16 @@
 from collections.abc import Hashable
 from dataclasses import dataclass
-from typing import (
-    Optional, TypeVar, Any, Type, Dict, Union, Iterator, Tuple, Set
-)
+from typing import Optional, TypeVar, Any, Type, Dict, Union, Iterator, Tuple, Set
 
 from dbt.contracts.graph.compiled import CompiledNode
 from dbt.contracts.graph.parsed import ParsedSourceDefinition, ParsedNode
 from dbt.contracts.relation import (
-    RelationType, ComponentName, HasQuoting, FakeAPIObject, Policy, Path
```

Comment:
There are a lot of these as well.  I'm a big fan of this change as it's much easier to see list membership when scanning up/down vs left/right (for me anyway)

Reply:
I'm not such a big fan of this because in some cases it's going to make the import headers pretty long. But I shrug...